src/cryptonote_basic/account.h

```diff
@@ -59,7 +59,7 @@ namespace cryptonote
   {
   public:
     account_base();
-    crypto::secret_key generate(const crypto::secret_key& recovery_key = crypto::secret_key(), bool recover = false, bool two_random = false);
+    crypto::secret_key generate(const crypto::secret_key& recovery_key = crypto::secret_key(), bool recover = false, bool two_random = false, bool from_legacy16B_lw_seed = false);
```

Comment:
I think this commit should be squashed to the first one so that it becomes a complete patch.

Reply:
Squashed